kubernetes/omsagent.yaml

```diff
@@ -379,13 +379,19 @@ spec:
               memory: 225Mi
           env:
             # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
+            - name: AKS_CLUSTER_NAME
```

Comment:
do you know where we use this and why this is needed ?

Reply:
Double checking this, looks like this was added long ago and the history until june2020 on this file doesnt tell me who added this. In June, this file was moved and has existed even before that. But looks like we are not using this, so removing this env var.